lib/ship.rb

```diff
@@ -0,0 +1,17 @@
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = length
+  end
+
+  def sunk?
+    @health <= 1
```

Comment:
@health <= 0 #health of 1 is not sunk in specifications

Reply:
Ah good catch!